packages/amplify_datastore/ios/Classes/types/hub/FlutterHubElement.swift

```diff
@@ -50,7 +50,7 @@ public struct FlutterHubElement {
                 modelName: modelName
             )
             self.version = hubElement.version
-            let serializedData = model["serializedData"] as? [String: Any] ?? [:]
+            let serializedData = model as? [String: Any] ?? [:]
             self.deleted = serializedData["_deleted"] as? Bool ?? false
```

Comment:
Shouldn't be needed anymore since `self.model` is already `[String: Any]`


```suggestion
            self.deleted = self.model["_deleted"] as? Bool ?? false
```